static/css/event-docs.less

```diff
@@ -0,0 +1,150 @@
+                    <h1 class="summary_heading">Mobile Financial Services</h1>
+                    <div class="event-meta h-event">
+                      <p class="event-meta_address h-adr">
+                        <span class="event-meta_city p-locality">San Francisco</span>
+                        <span class="event-meta_state p-region">CA</span>
+                        <span class="event-meta_venue p-extended-address">Super Dome</span>
+                      </p>
+                      <time class="event-meta_datetime dt-start" datetime="2014-01-21 2:30 PM CST">
+                        <span class="event-meta_date">1/21/2014</span>
+                        <span class="event-meta_time">2:30 PM CST</span>
+                      </time>
+                    </div>
+                </header>
+                <p class="summary_text">Every year, millions of American families buy a car – and it will be one of the most significant purchases they make. One key priority for us is protecting consumers from the silent pickpocket of discrimination.</p>
+              </a>
+              <footer>
+                <div class="tags tags__hide-header">
+                  <h4 class="tags_header u-visually-hidden">Topics:</h4>
+                  <ul class="tags_list">
+                    <li class="tags_tag">
+                      <a href="#" class="tags_link">
+                        <span class="tags_bullet" aria-hidden="true">&bull;</span> Students
+                      </a>
+                    </li>
+                    <li class="tags_tag">
+                      <a href="#" class="tags_link">
+                        <span class="tags_bullet" aria-hidden="true">&bull;</span> Consumers
+                      </a>
+                    </li>
+                    <li class="tags_tag">
+                      <a href="#" class="tags_link">
+                        <span class="tags_bullet" aria-hidden="true">&bull;</span> Consumers
+                      </a>
+                    </li>
+                  </ul>
+                </div>
+              </footer>
+              </div>
+            </div>
+        </article>
+  tags:
+    - cfgov-event-docs
+*/
+
+.event-post-preview {
+    border-top: 1px solid @gray-50;
+    padding-top:  1em;
+    
+    .media-object_image-container {
+         .respond-to-max(@mobile-max {
+            display: none;
+        });
+    }
+}
+
+.summary {
+    
+    .calendar-icon {
+        .cf-calendar-icon( attr(data-month), attr(data-day), @gray-50, @white, 1);
+        float: left;
+        margin: 0 (@grid_gutter-width/2) 0 0;
+    }
+
+    .event-meta {
+        color: @black;
+    }
+
+    .event-meta_address:before {
+        content: none;
+        margin: 0px;
+    }
+
+    .event-meta_venue:after {
+        content: "\2014";
+        margin-left: .5em;
+
+        .respond-to-max(@mobile-max {
+            content: none;
+        });
+    }
+
+    .summary_header {
```

Comment:
Do this and the `summary` prefixed items below need to be nested under `.summary`? They already include that in the block.